src/components/ui/Select/Select.tsx

```diff
@@ -87,7 +87,7 @@ const SelectItem = forwardRef<
     ref={ref}
     className={cn(
       'relative flex items-center',
-      'w-full rounded-sm text-sm outline-none  focus:bg-gray-light',
+      'w-full rounded-sm text-base font-normal outline-none focus:bg-gray-light',
```

Comment:
Could you please add the class `data-[state=checked]:bg-gray-light` to highlight the selected item?
The usage of this class is mentioned here:
https://www.radix-ui.com/primitives/docs/components/select#item

As the `className` may get longer, please consider the style mentioned here:
https://github.com/nishiki-tech/nishiki-frontend/blob/develop/docs/STYLEGUIDE.md#css-class-management

Reply:
The select items should also have a 6px of border-radius (= `rounded-md`), as per the Figma desing.
<img width="600" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/883f0e80-4bcc-4543-b19e-54acfe85eacf">

Reply:
Thanks, I've grouped the classes based on:
- Position and alignment
- Size and shape
- Padding and cursor
- Colors